engine/kv_engine.hpp

```diff
@@ -68,6 +73,7 @@ class KVEngine : public Engine {
 
     alignas(64) uint64_t newest_restored_ts = 0;
     PendingBatch *persisted_pending_batch = nullptr;
+    std::unordered_map<uint64_t, int> visited_skiplist_ids_;
```

Comment:
maybe vesited_skiplist_in_recovery? and no "_" in the end for public memble.

Reply:
ok